ledger/ledger-api-tests/tool/BUILD.bazel

```diff
@@ -103,7 +103,9 @@ conformance_test(
             cert-chain-file=$(rootpath //ledger/test-common/test-certificates:server.crt)
             trust-collection-file=$(rootpath //ledger/test-common/test-certificates:ca.crt)
             private-key-file=$(rootpath //ledger/test-common/test-certificates:server.pem.enc)
-            secrets-url="https://raw.githubusercontent.com/digital-asset/daml/main/ledger/test-common/files/server-pem-decryption-parameters.json"
+            #temporarily use decryption parameters from a gist to migrate to the new AES/GCM/NoPadding encryption
+            #secrets-url="https://raw.githubusercontent.com/digital-asset/daml/release/2.4.x/ledger/test-common/files/server-pem-decryption-parameters.json"
+            secrets-url="https://raw.githubusercontent.com/digital-asset/daml/aes-gcm/ledger/test-common/files/server-pem-decryption-parameters.json"
```

Comment:
That feels really brittle. Do we need this to be a web url? Can't we have a local file instead?

Reply:
Also this will break as soon as this branch is merged; perhaps a commit link instead of a branch link would work better?